code/Control/UserDefinedFormController.php

```diff
@@ -364,10 +364,10 @@ public function process($data, $form)
                     if ($submittedFormField && trim($submittedFormField->Value)) {
                         $email->setSubject($submittedFormField->Value);
                     } else {
-                        $email->setSubject($recipient->EmailSubject);
+                        $email->setSubject(SSViewer::execute_string($recipient->EmailSubject, $mergeFields));
```

Comment:
Looks good, but is it protected from HTML and injection? e.g. `Subject: How to <bold>template</bold> subjects` - tags in this case should be stripped, etc. `Subject: How to template subjects`
Thankfully the PHP method to mail should set subjects safely, but if not then injection is also possible. Perhaps a few tests also including "bad cases" would be ideal here :)

Reply:
I don't think any of the module's editable form fields allow the user to enter HTML, do they?

Reply:
Yeah, that's a good point :)